### ISSUE TITLE
Invisible Mobs?

### ISSUE THREAD

**Issue by jhth13rteen** [TheIllusiveC4/Champions#41]
#### Some mobs are invisible.
Sorry if this is already been posted, did a search for any issue with the "invisible tag" and found nothing, also checked comments at curse forge and no mention of this, so it might be a rare or weird problem.

Went to hunting Dim and a baby skeleton archer and a witch was totally invisible and it looked like I couldnt even hit them. This is with the MC Eternal Modpack. This same exact behavior was also observed during a YouTuber stream (Direwolf20) at the 11:56 mark. https://www.youtube.com/watch?v=j2cxAZvo3yQ

#### Versions:
* Champions: champions-1.12.2-1.0.11.3.jar
* Forge: 14.23.5.2847

#### Steps to Reproduce:
1. Play game with mod loaded.
2. Eventually some hostile mobs will be invisible.
3. ...Profit???
...

**Comment by TheIllusiveC4** [comment]
Can you reproduce this issue with only Champions loaded? Also, how do you know this is a Champions issue? Since I don't do anything with invisibility or mob rendering and it doesn't look like the mobs that killed Direwolf are champion mobs, I'm not sure where this issue points to Champions being the cause.

**Comment by jhth13rteen** [comment]
I'm thinking its a mod conflict with Primitive Mobs mod. I am currently doing some testing. I know Primitive Mobs has a coremod that does some funky things re: mobs for Primitive Mobs, so I will keep checking this out and report my findings. 

I did find a workaround however for whoever reads this. If you encounter untargetable invisible and otherwise immortal mobs, just log out (not completely, just to Minecraft Title Screen with the moving background) and log back in. For some reason it appears "OnMobRender" either when the mob spawns near you or entering a portal where mobs are or spawn, it bugs them out. I'll report more findings in a bit. Thanks for the quick response C4 and stay safe!

**Comment by jhth13rteen** [comment]
Okay.. After reading the Primitive Mobs Issues a bit with the "Invisible" tag, it seems this mod has lots of issues with blocks or mobs being invisible, so it has to be that mod instead. Sorry for the confusion and it didnt dawn on me it might be Primitive Mobs, the reason I thought it was this mod was because the issue didnt start happening until after I installed it, but that could just have been a weird coincidence. Going to close this issue and refer to here: https://github.com/Daveyx0/PrimitiveMobs/issues?q=Invisible for people who have the same question as I did. Thanks for your time, sorry for wasting it. <3
